test/e2e/inspect_test.go

```diff
@@ -289,4 +289,99 @@ var _ = Describe("Podman inspect", func() {
 		Expect(baseJSON[0].HostConfig.SecurityOpt).To(Equal([]string{"label=type:spc_t,label=level:s0", "seccomp=unconfined"}))
 	})
 
+	It("podman inspect pod", func() {
+		podName := "testpod"
+		create := podmanTest.PodmanNoCache([]string{"pod", "create", "--name", podName})
+		create.WaitWithDefaultTimeout()
+		Expect(create.ExitCode()).To(Equal(0))
+
+		inspect := podmanTest.Podman([]string{"inspect", podName})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect.ExitCode()).To(Equal(0))
+		Expect(inspect.IsJSONOutputValid()).To(BeTrue())
+		podData := inspect.InspectPodArrToJSON()
+		Expect(podData[0].Name).To(Equal(podName))
+	})
+
+	It("podman inspect pod with type", func() {
+		podName := "testpod"
+		create := podmanTest.PodmanNoCache([]string{"pod", "create", "--name", podName})
+		create.WaitWithDefaultTimeout()
+		Expect(create.ExitCode()).To(Equal(0))
+
+		inspect := podmanTest.Podman([]string{"inspect", "--type", "pod", podName})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect.ExitCode()).To(Equal(0))
+		Expect(inspect.IsJSONOutputValid()).To(BeTrue())
+		podData := inspect.InspectPodArrToJSON()
+		Expect(podData[0].Name).To(Equal(podName))
+	})
+
+	It("podman inspect latest pod", func() {
+		podName := "testpod"
+		create := podmanTest.PodmanNoCache([]string{"pod", "create", "--name", podName})
+		create.WaitWithDefaultTimeout()
+		Expect(create.ExitCode()).To(Equal(0))
+
+		inspect := podmanTest.Podman([]string{"inspect", "--type", "pod", "--latest"})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect.ExitCode()).To(Equal(0))
+		Expect(inspect.IsJSONOutputValid()).To(BeTrue())
+		podData := inspect.InspectPodArrToJSON()
+		Expect(podData[0].Name).To(Equal(podName))
+	})
+	It("podman inspect latest defaults to latest container", func() {
+		podName := "testpod"
+		pod := podmanTest.PodmanNoCache([]string{"pod", "create", "--name", podName})
+		pod.WaitWithDefaultTimeout()
+		Expect(pod.ExitCode()).To(Equal(0))
+
+		inspect1 := podmanTest.Podman([]string{"inspect", "--type", "pod", podName})
+		inspect1.WaitWithDefaultTimeout()
+		Expect(inspect1.ExitCode()).To(Equal(0))
+		Expect(inspect1.IsJSONOutputValid()).To(BeTrue())
+		podData := inspect1.InspectPodArrToJSON()
+		infra := podData[0].Containers[0].Name
+
+		inspect := podmanTest.Podman([]string{"inspect", "--latest"})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect.ExitCode()).To(Equal(0))
+		Expect(inspect.IsJSONOutputValid()).To(BeTrue())
+		containerData := inspect.InspectContainerToJSON()
+		Expect(containerData[0].Name).To(Equal(infra))
+	})
+
+	It("podman inspect network", func() {
+		name, path := generateNetworkConfig(podmanTest)
+		defer removeConf(path)
+
+		session := podmanTest.Podman([]string{"network", "inspect", name, "--format", "{{.cniVersion}}"})
```

Comment:
Should this be `network inspect` or just inspect?

Reply:
inspect, thanks